generated/nidaqmx/constants.py

```diff
@@ -993,12 +1017,15 @@ class UsageTypeCI(Enum):
     POSITION_ANGULAR_ENCODER = 10360  #: Angular position measurement using an angular encoder.
     POSITION_LINEAR_ENCODER = 10361  #: Linear position measurement using a linear encoder.
     TIME_GPS = 10362  #: Timestamp measurement, synchronizing the counter to a GPS receiver.
+    TIMESTAMP = 15986  #: Timestamp measurement.
+    CHANNEL_TYPE_INDICATOR = 16030
```

Comment:
This looks like an internal value. Can you remove?

Reply:
Removed.